runtime/include/tt/runtime/runtime.h

```diff
@@ -13,6 +13,12 @@
 
 namespace tt::runtime {
 
+const DeviceRuntime &getCurrentRuntime();
```

Comment:
Sorry, it's a bit nit picky, but this should return by value, `DeviceRuntime getCurrentRuntime();`.